src/client/components/logout/logout.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { logoutIcon } from '../../assets/icons/icons';
+import './logout.css';
+import Button from '../button';
+
+class Logout extends Component {
+    SignOut = async () => {
+      const { sound, voice } = this.props.audioData;
+      const { history } = this.props;
+      if (sound) {
+        sound.stop();
+        voice.stop();
+      }
+      history.push('/');
+    }
+
+    render() {
+      const LogoutButton = (
+        <Button
+          className="ButtonStyleTemplate"
+          icon={logoutIcon}
+          value="Logout"
+          onClick={this.SignOut}
+        />
+      );
+      return (
+        <div>
```

Comment:
Do you actually need this div here?

Reply:
@tonypntsrn 
![image](https://user-images.githubusercontent.com/40705411/61462342-234f9380-a97b-11e9-975f-3e4917fb58ae.png)
reworked on such an option

Reply:
don't need this "div"
reworked for the best option